compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeHashVisitor.java

```diff
@@ -475,7 +475,8 @@ public Integer visit(BRecordType type) {
         }
         List<Integer> fieldsHashes = getFieldsHashes(type.fields);
         List<Integer> typeInclHashes = getTypesHashes(type.typeInclusions);
-        Integer hash = hash(baseHash(type), type.sealed, fieldsHashes, typeInclHashes, visit(type.restFieldType));
+        Integer hash = hash(baseHash(type), type.sealed, fieldsHashes, typeInclHashes, visit(type.restFieldType),
```

Comment:
This line is changed in `stage-slbeta1` branch. Therefore, this can conflict. Better if we can wait for `stage-slbeta1` to sync with master and update the PR.
https://github.com/ballerina-platform/ballerina-lang/blob/stage-slbeta1/compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeHashVisitor.java#L479